src/main/java/org/prnhs/javaee/swim/web/ProgramController.java

```diff
@@ -15,26 +17,32 @@
 @RequestMapping("/program")
 public class ProgramController {
     
+    private static final Logger LOGGER = LoggerFactory.getLogger(ProgramController.class);
+    
     @Autowired
     private ProgramService programService;
     
     @RequestMapping(value="/", method = RequestMethod.POST, consumes=MediaType.APPLICATION_JSON_VALUE, produces=MediaType.APPLICATION_JSON_VALUE)
     public ProgramDto save(@RequestBody ProgramDto dto){
+        LOGGER.debug("Save method called in ProgramController class");

```

Comment:
I only have 2 fields at my programDto class. Id and Objective. And since I get a null id when POST is executed, I will print only the Objective field